src/main/java/com/example/exercise1/controller/HelloWorldController.java

```diff
@@ -36,4 +42,14 @@ public long getRemainingTime() {
         return timeMilli;
     }
 
+    @PostMapping(
+            value = "/api/sayHi"
+//            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE}
```

Comment:
What happened if we uncommented this line?
` consumes = {MediaType.APPLICATION_JSON_VALUE}`

Reply:
nothing will happen, since the returned data is already JSON, it will be redundant